### ISSUE TITLE
[EN] [Quest] [Blade's Edge Mountains] Fire At Will!

### ISSUE THREAD

**Issue by Dexobox** [Atlantiss/BugTracker#2507]
This quest involves taking control of the [Death's Door Fel Cannon] (http://www.wowhead.com/npc=51612/deaths-door-fel-cannon)  at [Death's Door] (http://www.wowwiki.com/Death's_Door) in Blade's Edge Mountians.

The bug for this quest is : 

1. The **"South Warp-Gate Destroyed"** objective cannot be completed as the [Death's Door Fel Cannon](http://www.wowhead.com/npc=51612/deaths-door-fel-cannon) __(Southern Cannon: 65, 68)__ close to the South Warp Gate does not attack the warp gate or the imps that are in the warp gate

2. Also for both the warp gates the imps are supposed to come out and attack the character doing the quest. The [Unstable Fel-Imp](http://www.wowhead.com/npc=22474/unstable-fel-imp) are bugged.Also the [Void Hound](http://www.wowhead.com/npc=22500/void-hound) npc are missing.

Wowhead Link: [Fire At Will!](http://www.wowhead.com/quest=10911/fire-at-will)
Youtube Video: [Fire At Will!](https://www.youtube.com/watch?v=h1wUajrO00c)

**Comment by LordKnightCecilo** [comment]
This quest worked for me, but only after the server restarted, maybe whatever counts the progress dies and doesn't re spawn properly?

**Comment by Illusionist87** [comment]
Friend of mine did this quest a few days ago, but now cannons needed for the quest are missing. So the quest is impossible to complete at the moment.

**Comment by illidan810** [comment]
I fired 10 shots at each gate and still nothing. Thus quest is still broken

**Comment by rwystop** [comment]
and i can't even shoot !!! i cant enter to this cannon !

**Comment by siwy92** [comment]
stil not work

**Comment by Klickeazy** [comment]
As of date, the North Gate goes down. The South gate has a place holder who takes the shots from the fel cannon but the full 3 mins of shots dont seem to take it down. I dont think this was any time soon after a server restart so that would have to be checked. I think the issue is with the place holder being reset after being taken down

**Comment by Thorghar** [comment]
Repair this quest plz.
**_STILL NOT WORKING_ !!!!!!**

**Comment by Hallieth** [comment]
Yup, still bugged, South gate could not be taken down no matter how long I have tried. :)

**Comment by aendry** [comment]
still not working

**Comment by m3inbl0ck** [comment]
Please fix this. Cenarion Expedition rep is hard to obtain as it is.

**Comment by cyberknight1993** [comment]
Confirmed still not working. In my case North Gate cannon became my minion but I was not in control of it, did not have any cannon abilities to use, imps inside gate did not do anything and wasn't targetable.

As for South Gate cannon, I couldn't use item on it at all, said the cannon is missing while I stood near it.
